### PR TITLE
Redis flood flush command

### PR DESCRIPTION
**Pull request by AronNovak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -264,3 +264,29 @@ It saves you time and disk space by sending requests to your development environ
...
+```
+
+It purges all the entries from `test` environment.
+
+```bash
+ddev pantheon-flood-flush test 193.165.2.3
+ddev ddev-flood-flush 193.165.2.3
+```
+
+It purges entries related to `193.165.2.3` from `test` environment, or alternatively from DDEV's own Redis.
```
Comment (amitaibu) [inline README.md:292]:
```suggestion
Purges entries related to IP `193.165.2.3` from Pantheon's `test` environment, or alternatively from DDEV's own Redis.
```

File: README.md [hunk 2]
```diff
@@ -264,3 +264,29 @@ It saves you time and disk space by sending requests to your development environ
 to the production environment and making a copy of the production file in your development site.
 
 Configure the origin path at `/admin/config/system/stage_file_proxy`.
+
+### Flood control
+
+As the project uses Redis, it is not possible to use the SQL console to reset flood table.
+There's a custom DDEV command to help with that.
+Usages:
```
Comment (amitaibu) [inline README.md:272]:
```suggestion
There's a custom DDEV command to help with that. Usages:
```

File: web/sites/default/settings.ddev.php [hunk 3]
```diff
@@ -66,6 +66,7 @@
 $settings['redis.connection']['host']      = 'redis';  // Your Redis instance hostname.
 $settings['cache']['default'] = 'cache.backend.redis';
 $settings['container_yamls'][] = 'modules/contrib/redis/redis.services.yml';
+$settings['container_yamls'][] = 'modules/contrib/redis/example.services.yml';
```
Comment (amitaibu) [inline web/sites/default/settings.ddev.php:69]: Is this one correct?

Comment (AronNovak) [reply web/sites/default/settings.ddev.php:69]: Yep, that''s the correct one, we should drop the previous line:
```
Either include the default example.services.yml from the module, which will
replace all supported backend services (that currently includes the cache tags
checksum service and the lock backends, check the file for the current list)
or copy the service definitions into a site specific services.yml.

    $settings['container_yamls'][] = 'modules/redis/example.services.yml';
```

Comment (AronNovak) [reply web/sites/default/settings.ddev.php:68]: Hm, if Redis module is not enabled, we need that live, Travis says so :)

Comment (amitaibu) [reply web/sites/default/settings.ddev.php:69]: Ok, so we should leave as is? If so, please merge

**Pull request merged**